### DISCUSSION TITLE
raspberry pi blank screen

### DISCUSSION THREAD

**Discussion by arashtotti88** [neutrinolabs/xrdp#2129] [Q&A]
Hi

I have had this problem for so long and there isn't any direct tutorial for how to fix it 

I install xrdp just after reinstalling raspberry pi os but still I'm facing a blank dark screen after login

sudo apt install xrdp 
sudo adduser xrdp ssl-cert  

can anyone just help me and millions of users with very direct soloution 

appreciate it

**Comment by matt335672** [comment]
Hi @arashtotti88 

The one which often causes this is that the user is logged in at the console as `pi`, and then tries to log in over xrdp with the same username. It's common on pi machines as the console is often auto logged-in.

You can't have the same user logged in at the console and over xrdp at the same time. That's a `systemd --user` limitation, and nothing to do with us.

Is that it? If not we'll start looking at log files.

**Reply by arashtotti88** [reply]
thanks for explanation 
but what is the solution 

where is the direct tutorial for fixing the issue?

**Reply by matt335672** [reply]
The simplest thing to do is to add another user to the pi with the `adduser` command and use that for xrdp.

@Linden1 makes a great point below. You'll need to do this too to avoid the problems in #2060

**Comment by Linden1** [comment]
Edit /etc/X11/xrdp/xorg.conf

Find :
Option "DRMDevice" "/dev/dri/renderD128"

And replace with:
Option "DRMDevice" ""

Reboot and you're good to go.

**Reply by arashtotti88** [reply]
unfortunately after doing this still black screen issue is remained

**Reply by matt335672** [reply]
Have you set up a different user for xrdp yet?

**Reply by arashtotti88** [reply]
yes I, did

with another user which I created it works fine, but still have problems with pi user and root user

**Reply by matt335672** [reply]
If you really want to use the `pi` user to log in remotely you'll have to make sure the console is completely logged out. Disabling auto-logins might be a good idea.

By default root logins are not allowed over xrdp.

The best solution to both of these is to use your separate user for xrdp and also add it to the `sudo` group so you can run root commands.

**Reply by arashtotti88** [reply]
thanks for great and direct help
